README.md

```diff
@@ -70,6 +70,24 @@ Pure solidity repository of rhino.fi's contracts. No dependencies are included a
 - MultiSig - N/A
 - Timelock - N/A
 
+#### Manta
+- Bridge [0x2B4553122D960CA98075028d68735cC6b15DeEB5](https://manta-pacific.calderaexplorer.xyz/address/0x2B4553122D960CA98075028d68735cC6b15DeEB5)
+- Cross-Swap wallet - N/A
+- MultiSig - N/A
+- Timelock - N/A
+
+#### Starknet
+- Bridge [0x0259fec57cd26d27385cd8948d3693bbf26bed68ad54d7bdd1fdb901774ff0e8](https://starkscan.co/contract/0x0259fec57cd26d27385cd8948d3693bbf26bed68ad54d7bdd1fdb901774ff0e8#portfolio)
```

Comment:
Do we want the link to point to the portfolio or to the default page? (https://starkscan.co/contract/0x0259fec57cd26d27385cd8948d3693bbf26bed68ad54d7bdd1fdb901774ff0e8)